csharp/src/Ice/Communicator-LocatorManager.cs

```diff
@@ -76,17 +76,9 @@ public void Response(LocatorInfo locatorInfo, IObjectPrx? proxy)
 
             public void Exception(LocatorInfo locatorInfo, System.Exception exc)
```

Comment:
`locatorInfo` is no longer used, can we remove this parameter?

Reply:
Fixed.